mysql-test/suite/perfschema/t/show_sanity.test

```diff
@@ -501,6 +501,7 @@ insert into test.sanity values
   ("JUNK: GLOBAL-ONLY", "I_S.SESSION_VARIABLES", "INNODB_PAGE_CLEANER_DISABLED_DEBUG"),
   ("JUNK: GLOBAL-ONLY", "I_S.SESSION_VARIABLES", "INNODB_PAGE_HASH_LOCKS"),
   ("JUNK: GLOBAL-ONLY", "I_S.SESSION_VARIABLES", "INNODB_PAGE_SIZE"),
+  ("JUNK: GLOBAL-ONLY", "I_S.SESSION_VARIABLES", "INNODB_PARALLEL_DOUBLEWRITE_PATH"),
```

Comment:
Hm, not sure, innodb_log_checksums is also there but it's definitely not a 5.6 variable